src/Keyboard.js

```diff
@@ -1,229 +1,331 @@
-import React, {PureComponent} from 'react';
-import PropTypes from 'prop-types';
-import KeyboardButton from './KeyboardButton';
-
-import LatinLayout from './layouts/LatinLayout';
-import CyrillicLayout from './layouts/CyrillicLayout';
-import SymbolsLayout from './layouts/SymbolsLayout';
-
-import BackspaceIcon from './icons/BackspaceIcon';
-import LanguageIcon from './icons/LanguageIcon';
-import ShiftIcon from './icons/ShiftIcon';
-
-export default class Keyboard extends PureComponent {
-	static propTypes = {
-		leftButtons: PropTypes.arrayOf(PropTypes.node),
-		rightButtons: PropTypes.arrayOf(PropTypes.node),
-		inputNode: PropTypes.any.isRequired,
-		onClick: PropTypes.func,
-		isFirstLetterUppercase: PropTypes.bool,
-		layouts: PropTypes.arrayOf(PropTypes.shape({
-			symbolsKeyValue: PropTypes.string,
-			layout: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
-		})),
-	};
-
-	static defaultProps = {
-		leftButtons: [],
-		rightButtons: [],
-		isFirstLetterUppercase: false,
-		layouts: [CyrillicLayout, LatinLayout],
-	};
-
-	state = {
-		currentLayout: 0,
-		showSymbols: false,
-		uppercase: this.isUppercase(),
-	}
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+	value: true
+});
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
```

Comment:
It doesn't look like source code

Reply:
Sorry, I don't know what happened, I was in a rush. I'll check and fix.

Reply:
Now it should be ok. Sorry again :)